validator/tests/test_scheduler/test_predecessor_tree.py

```diff
@@ -109,20 +109,6 @@ def test_predecessor_tree(self):
         #           i:
         #             c: Readers: [5]
 
-        self.assert_rwc_at_addresses({
```

Comment:
These tests should be rewritten to work with the new API so that we do not lose coverage instead of being deleted.

Reply:
Those assertions check specific implementation details of the tree data structure, and so would have to be rewritten for any change to the tree, including this PR.

Reply:
Removing these tests are fine: We shouldn't be using unit tests to verify internal state.

Reply:
I added commit message to reflect that change.